actstream/models.py

```diff
@@ -24,7 +26,7 @@ class Follow(models.Model):
     content_type = models.ForeignKey(
         ContentType, on_delete=models.CASCADE, db_index=True
     )
-    object_id = models.CharField(max_length=255, db_index=True)
+    object_id = models.UUIDField(max_length=255, db_index=True, default=uuid.uuid4)
```

Comment:
@Vaibhav015 i think we do not need to add `default=uuid.uuid4`

Reply:
Also, why have we given `max_length` in uuid field ? I mean do not require this ?

Reply:
As per docs, It's recommended to use ```default``` in UUIDField @veerat-pdgt 
Ref Docs: https://docs.djangoproject.com/en/3.0/ref/models/fields/#uuidfield
cc @vikalpj 
Let me know your thoughts @veerat-pdgt @vikalpj  🙂

Reply:
@vaibhav-pdgt The case mentioned in doc is only applicable when we have PK for uuid.  else default should always be based on the usecase. :)

Reply:
yeah, thanks @vikalpj

Reply:
fixes done